### PR TITLE
Improve RetroArch

### PR DESCRIPTION
**Pull request by adamscott**
#### Editing an existing cask
- [x] Commit message includes cask’s name (and new version, if applicable).
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

---

Rename retroarchcg.rb to retroarch-cg.rb

Upgrade RetroArch CG.app to v1.3.3
- Correct the url pattern
- Rename the cask
- Rename the .app for clarity

Add RetroArch.app v1.3.3

---

As two versions of RetroArch for MacOS X [exists](http://buildbot.libretro.com/stable/1.3.3/apple/osx/x86_64/), it is now reflected by this commit. `retroarch` cask is the non-CG version. `retroarch-cg` now is.

### CODE REVIEW

File: Casks/retroarch-cg.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask 'retroarch-cg' do
+  version '1.3.3'
+  sha256 '59410f57518353076fa17afa8b3c1a8076c5fd25829f52f2ff3712bdb0513abe'
+
+  url "https://buildbot.libretro.com/stable/#{version}/apple/osx/x86_64/RetroArch_CG.dmg"
+  name 'RetroArch CG'
+  homepage 'http://www.libretro.com/'
+  license :gpl
+
+  # Renamed for clarity: 2 versions of RetroArch exists for MacOS X.
```
Comment (vitorgalvao) [inline Casks/retroarch-cg.rb]: Comment needs to be exact: `# Renamed to avoid conflict with retroarch.`

File: Casks/retroarch-cg.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  # There is RetroArch (non-CG) and this version.
```
Comment (vitorgalvao) [inline Casks/retroarch-cg.rb]: Remove this second line.

**Comment by vitorgalvao** [comment]
In the future, please submit separate pull requests for each cask, as more often than not this type of pull request brings problems with one or more of them, and it halts the inclusion of the other ones.

**Comment by adidalal** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 7b803e14942aa51eb705f8716a4861e385562050. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by adamscott** [comment]
No problem! Thanks to you.

But, I'm curious about the fixes made in 7b803e1. What needed to change? I checked your commit and nothing seems different.

**Comment by adidalal** [comment]
Squashed commits

**Pull request closed without merging**